queue/bucket.js

```diff
@@ -63,6 +63,14 @@ Bucket.prototype.createPutUrl = function(prefix, options) {
 };
 
 /**
+ * Create an unsigned GET URL
+ */
+Bucket.prototype.createGetUrl = function(prefix) {
+  assert(prefix,                "prefix must be given");
+  return `${this.s3.endpoint.href}${this.bucket}/${prefix}`;
```

Comment:
is this for real?  Google searches suggest yes.  This makes me happy to see in JS.